website/docs/feature-flag-tutorials/react/examples.md

```diff
@@ -82,7 +82,7 @@ Canary deployments are a safer and more gradual way to make changes in software
 Unleash has a few ways to help manage canary deployments for React apps at scale:
 
 
-* Using a [gradual rollout](/reference/activation-strategies#gradual-rollout) (which we implemented in a previous section) would be a simple use case but would reduce the amount of control you have over who gets the new feature.
+* Using a [gradual rollout](/reference/activation-strategies#gradual-rollout) (which we [implemented above](#gradual-rollouts-for-react-apps)) would be a simple use case but would reduce the amount of control you have over who gets the new feature.
```

Comment:
I suggest avoiding the word "above" here because blind users don't necessarily have any directional concept of text. Microsoft mention it in their writing guide [in reference to screen content](https://learn.microsoft.com/en-us/style-guide/accessibility/writing-all-abilities#use-content-structure-and-location-to-communicate). I feel like I've heard the same about text in general, but I'm not sure. Happy to be wrong on this.


```suggestion
* Using a [gradual rollout](/reference/activation-strategies#gradual-rollout) (which we [implemented in a previous section](#gradual-rollouts-for-react-apps)) would be a simple use case but would reduce the amount of control you have over who gets the new feature.
```